python/packages/ai/teams/ai/promptsv2/data_source_section.py

```diff
@@ -0,0 +1,69 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from typing import List
+
+from botbuilder.core import TurnContext
+
+from ...state import Memory
+from ..data_sources import DataSource
+from ..tokenizers import Tokenizer
+from .message import Message
+from .prompt_functions import PromptFunctions
+from .prompt_section_base import PromptSectionBase
+from .rendered_prompt_section import RenderedPromptSection
+
+
+class DataSourceSection(PromptSectionBase):
+    """
+    A section that renders a data source to a prompt.
+    """
+
+    _data_source: DataSource
+
+    def __init__(self, dataSource: DataSource, tokens: float):
```

Comment:
`dataSource` should probably be `data_source` to match casing

Reply:
Good catch! Since pushing new commits will dismiss your approvals, I will merge this PR first and create a new PR which resolves this comment as well as other changes I'm going to make today.
Will paste the new PR link here once I created it.

Reply:
Fixed in https://github.com/microsoft/teams-ai/pull/1228/files#diff-66d64d25653fcf6abb1560a77496ff35bbdb263becf94b6c51eee40b85e7b9ab